tox.ini

```diff
@@ -56,7 +72,7 @@ basepython = python2.7
 deps = {[testenv]basedeps}
        nosegae
 commands =
-    nosetests --with-gae --gae-application=tests/data --logging-level=INFO {posargs} tests/test_appengine.py
+    nosetests --with-gae --gae-application=tests/data --logging-level=INFO tests/test_appengine.py
```

Comment:
It was there for testing the nose command, so you can run ``tox -e gae -- -vs`` for example.